executor/src/witgen/mod.rs

```diff
@@ -50,8 +50,8 @@ impl<T, F> QueryCallback<T> for F where F: Fn(&str) -> Result<Option<T>, String>
 #[derive(Clone)]
 pub struct WitgenCallback<T> {
     analyzed: Rc<Analyzed<T>>,
-    fixed_col_values: Rc<Vec<(String, Vec<T>)>>,
-    query_callback: Arc<dyn QueryCallback<T>>,
+    pub fixed_col_values: Rc<Vec<(String, Vec<T>)>>,
+    pub query_callback: Arc<dyn QueryCallback<T>>,
```

Comment:
haha

Reply:
I guess just make all of them `pub` at this point?

Reply:
Or add getters. I think about when I clean up this PR :P